inspect.go

```diff
@@ -0,0 +1,166 @@
+package enmime
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"mime"
+	"net/textproto"
+	"strings"
+
+	"github.com/jhillyerd/enmime/internal/coding"
+	"github.com/pkg/errors"
+)
+
+func HumanHeadersOnly(b []byte) (map[string]string, error) {
```

Comment:
It doesn't look like we are actually limiting the returned headers to a human list, and we probably don't need to do that anyway.  Should we just call it `DecodeHeaders` instead?

Reply:
Now limited to a human list to reduce execution time, but still renamed to `DecodeHeaders` and added variadic second parameter for supplying additional header keys the consumer wants in the returned map.

Reply:
Since function signature has changed, I will resolve this conversation and look forward to new proposed changes on the latest commit.